homeassistant/components/media_player/trigger.py

```diff
@@ -0,0 +1,34 @@
+"""Provides triggers for media players."""
+
+from homeassistant.core import HomeAssistant, State
+from homeassistant.helpers.trigger import EntityTriggerBase, Trigger
+
+from . import ATTR_MEDIA_VOLUME_LEVEL, ATTR_MEDIA_VOLUME_MUTED
+from .const import DOMAIN
+
+
+class MediaPlayerMutedTrigger(EntityTriggerBase):
+    """Class for media player muted triggers."""
+
+    _domain: str = DOMAIN
+
+    def is_muted(self, state: State) -> bool:
+        """Check if the media player is muted."""
+        return (
+            state.attributes.get(ATTR_MEDIA_VOLUME_MUTED) is True
+            or state.attributes.get(ATTR_MEDIA_VOLUME_LEVEL) == 0
```

Comment:
What would be the alternative to not firing the trigger if the attributes are missing? What other behavior could be expressed?